.github/renovate.json5

```
@@ -25,6 +25,14 @@
       matchManagers: ["cargo"],
       enabled: false,
     },
+    {
+      // Create dedicated branches to update references to dependencies in the documentation.
+      matchPaths: ["docs/**/*.md"],
+      commitMessageTopic: "references to {{{depName}}}",
```

For clarity, can we mention the docs in the title?

```suggestion
      commitMessageTopic: "documentation references to {{{depName}}}",
```